tests/unit/test_connection_overrides.py

```diff
@@ -174,3 +174,41 @@ def test_synapse_modoverride(create_tmp_simulation_config_file):
         for nc_id, nc in enumerate(nclist):
             utils.check_netcon(src_gid, nc_id, nc, edges, selection, **kwargs)
             utils.check_synapse(nc.syn(), edges, selection, **kwargs)
+
+
+@pytest.mark.parametrize("create_tmp_simulation_config_file", [
+    {
+        "simconfig_fixture": "ringtest_baseconfig",
+        "extra_config": {
+            "target_simulator": "NEURON",
+            "node_set": "Mosaic",
+            "connection_overrides": [
+                {
+                    "name": "A2B",
+                    "source": "RingA",
+                    "target": "RingB",
+                    "spont_minis": 200,
+                    "synapse_configure": "%s.verboseLevel = 1"
+                }
+            ]
+        }
+    },
+], indirect=True)
+def test_spont_minis_simple(create_tmp_simulation_config_file):
```

Comment:
Can we check  the netcon object created by `self._spont_minis.create_on()`?  That should be stored in the `Nd.cvode.netconlist`.

Reply:
the netcon is always there. How do you check if there is a spont minis attached or not?

PS: yes, I know how to retrieve a netcon as I worked on `connection_overrides`

Reply:
Maybe something like the test for Replay netcons ? 
https://github.com/openbraininstitute/neurodamus/blob/ae27d40ac931e49cc122353f08d2111a7bb9c772/tests/unit/test_replay_manager.py#L175

Reply:
it does not work. netcon.weight[4] is 0. Even if clearly the `spont_minis` is attached. How do I know that it works?

because I see peaks with the spont minis and no peaks without

Reply:
also weight[4] is not documented. I think for the replay in particular it needs to pass something to the netcon and it puts the type there but I do not think it is a common feature in neurodamus

Reply:
btw, the weight vector is: `[100.0, 0.0, 0.0, 0.0, 0.0]`

Reply:
done!